OpenSim/Common/TimeSeriesTable.h

```diff
@@ -468,6 +477,31 @@ class TimeSeriesTable_ : public DataTable_<double, ETY> {
     void trimTo(const double& newFinalTime) {
         this->trim(this->getIndependentColumn().front(), newFinalTime);
     }
+    /**
+     * trim table to rows between start_index and last_index inclusively
```

Comment:
This comment will appear in the Doxygen documentation, so use full sentence case and grammar: "Trim the TimeSeriesTable  to the rows between start_index and last_index, inclusively.